src/components/music/Scale.js

```diff
@@ -98,6 +98,10 @@ export default class Scale {
     )
   }
 
+  getInstructionString() {
+    return "asteikon pohjasävel ja laatu"
```

Comment:
Tarkista `this.usedMode === "modes"` ja käytä moodeille järkevää muotoa tehtävänannosta